frontend/src/components/AddPostForm/AddPostForm.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Wrapper, Header, Icon, TitleInput, MessageInput, SubmitInput } from './AddPostForm.styled';
+
+const AddPostForm = () => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(event);
+  };
+
+  return (
+    <Wrapper>
+      <form onSubmit={handleSubmit}>
+        <Header>Add new post</Header>
+        <Icon>Icon</Icon>
+        <TitleInput type="text" defaultValue="Enter title..." />
```

Comment:
Czy to nie będzie `Label`?

Reply:
Racja, dobra nazwa

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

![image](https://user-images.githubusercontent.com/41823689/149670137-8c33f6a0-26ef-4b62-878c-98262369006a.png)